Library/Homebrew/livecheck/strategy/github_latest.rb

```diff
@@ -0,0 +1,59 @@
+# typed: false
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      # The {GithubLatest} strategy identifies versions of software at
+      # github.com by checking a repository's latest release page.
+      #
+      # GitHub URLs take a few differemt formats:
+      #
+      # * `https://github.com/example/example/releases/download/1.2.3/example-1.2.3.zip`
+      # * `https://github.com/example/example/archive/v1.2.3.tar.gz`
```

Comment:
a few repos I checked ([1](https://github.com/intel/intel-graphics-compiler/releases/latest), [2](https://github.com/NVIDIA/enroot/releases/latest)) have the same version in the path: `download/1.2.3` and `archive/1.2.3`